src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2538,39 +2538,39 @@ namespace rpc {
   };
 
   OXEN_RPC_DOC_INTROSPECT
-  // Performs a simple LNS lookup of a BLAKE2b-hashed name.  This RPC method is meant for simple,
+  // Performs a simple ONS lookup of a BLAKE2b-hashed name.  This RPC method is meant for simple,
   // single-value resolutions that do not care about registration details, etc.; if you need more
-  // information use LNS_NAMES_TO_OWNERS instead.
+  // information use ONS_NAMES_TO_OWNERS instead.
   //
   // Technical details: the returned value is encrypted using the name itself so that neither this
   // oxend responding to the RPC request nor any other blockchain observers can (easily) obtain the
   // name of registered addresses or the registration details.  Thus, from a client's point of view,
-  // resolving an LNS record involves:
+  // resolving an ONS record involves:
   //
   // - Lower-case the name.
   // - Calculate the name hash as a null-key, 32-byte BLAKE2b hash of the lower-case name.
-  // - Obtain the encrypted value and the nonce from this RPC call (or LNS_NAMES_TO_OWNERS); (encode
+  // - Obtain the encrypted value and the nonce from this RPC call (or ONS_NAMES_TO_OWNERS); (encode
   //   the name hash using either hex or base64.).
   // - Calculate the decryption key as a 32-byte BLAKE2b keyed hash of the name using the
   //   (unkeyed) name hash calculated above as the hash key.
   // - Decrypt (and verify) using XChaCha20-Poly1305 (for example libsodium's
   //   crypto_aead_xchacha20poly1305_ietf_decrypt) using the above decryption key and using the
   //   first 24 bytes of the name hash as the public nonce.
-  struct LNS_RESOLVE : PUBLIC
+  struct ONS_RESOLVE : PUBLIC
   {
-    static constexpr auto names() { return NAMES("lns_resolve"); }
+    static constexpr auto names() { return NAMES("ons_resolve"); }
```

Comment:
lokinet calls `lns_resolve` right now so mind the backwards compat on rpc method names, at least keep old methods as well as adding new ones for now.